### PR TITLE
Add OliveTin to Automation category

### PR DESCRIPTION
**Pull request by jamesread**
Hi folks. This PR is to add OliveTin to the Automation category. 

https://github.com/jamesread/OliveTin

I have checked all items in the PR template and as far as I can see, followed all the rules and guidance listed. 

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of
  - [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management),
  - [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics),
  - [staticgen.com](https://www.staticgen.com/)
  - [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] The `Language` tag is the main server-side requirement for the software - don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

Thanks so very much for your time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ _See also [Internet of Things (IoT)](#internet-of-things-iot)_
 - [MeTube](https://github.com/alexta69/metube) - Web GUI for youtube-dl, with playlist support. Allows downloading videos from dozens of websites. `AGPL-3.0` `Python/Nodejs/Docker`
 - [n8n](https://n8n.io/) - Free node based Workflow Automation Tool. Easily automate tasks across different services. ([Source Code](https://github.com/n8n-io/n8n)) `Apache-2.0` `Nodejs`
 - [nefarious](https://github.com/lardbit/nefarious) - Web application that automates downloading Movies and TV Shows. `GPL-3.0` `Python`
+- [OliveTin](https://github.com/jamesread/OliveTin) - OliveTin is a web interface for running Linux shell commands. ([Demo](https://www.youtube.com/watch?v=Ej6NM9rmZtk)) `AGPL-3.0` `Go`
```
Comment (nodiscc) [inline README.md:160]: `Demo` links should only be used for interactive demos, i.e. not video demonstrations
 (https://github.com/awesome-selfhosted/awesome-selfhosted/issues/2459). Please remove the demo link.

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jamesread) [reply README.md]: I understand, sorry for the confusion. I didn't see this in the rules but could have checked the other demo examples. Commit b5cbdea should address this.

**Comment by jamesread** [comment]
@nodiscc should be ready for re-review when your time allows. Thank you very much for maintaining this list.

**Review by nodiscc — APPROVED** [review]

**Comment by jamesread** [comment]
Hey @nodiscc , I saw you approved these changes, and I assume you've just not yet had the opportunity to do the merge? 

Just checking if there is anything else you'd like me to do - otherwise I will just be patient and wait for you to get some free time :)

**Pull request merged**